cantabular/client.go

```diff
@@ -84,9 +84,9 @@ func (c *Client) Checker(ctx context.Context, state *healthcheck.CheckState) err
 		"service": Service,
 	}
 	code := 0
-	urlHost := fmt.Sprintf("%s/v9/datasets", c.host)
+	hostPath := fmt.Sprintf("%s/v9/datasets", c.host)
```

Comment:
Sorry I think hostPath is still a bit ambiguous.. Really `url` is fine, it's a really common naming convention for this situation

Reply:
Adjusted to `reqURL` as per discussion elsewhere.